### DISCUSSION TITLE
页眉如何修改

### DISCUSSION THREAD

**Discussion by zw-yu** [sjtug/SJTUThesis#938] [提问] [answered]
希望将”上海交通大学学士学位论文“改为”上海交通大学学位论文“

**Comment by LogCreative** [comment, marked as answer]
如果只改页眉，在 `\maketitle` 后添加
```latex
\sjtusetup{info/zh/subject={上海交通大学学位论文}}
```

**Reply by zw-yu** [reply]
1、另外我想把封面的”上海交通大学学士学位论文“也改为”上海交通大学学位论文“。

2、我想在封面最后添加一项：申请学位层次：学士。

3、我想把封面上的“学科/专业”改为“学科/专业名称”

**Reply by LogCreative** [reply]
> 1、另外我想把封面的”上海交通大学学士学位论文“也改为”上海交通大学学位论文“。


将代码移至 \maketitle 之前。
